contrib/babelfishpg_tsql/src/multidb.c

```diff
@@ -924,6 +924,41 @@ rewrite_column_refs(ColumnRef *cref)
 static void
 rewrite_rangevar(RangeVar *rv)
 {
```

Comment:
Only rewriting the range variable is sufficient?
Can you please more tests for
- nested statements, 
- joins,
- when views are in SELECT, FROM, WHERE clause etc.